requirements/linting.txt

```diff
@@ -38,18 +30,12 @@ pathspec==0.10.1
     # via black
 platformdirs==2.5.2
     # via black
-pycodestyle==2.9.1
-    # via flake8
-pyflakes==2.5.0
-    # via flake8
+ruff==0.0.130
+    # via -r requirements/linting.in
 sniffio==1.3.0
     # via trio
 sortedcontainers==2.4.0
     # via trio
-tomli==2.0.1
```

Comment:
Are you sure `tomli` should be removed here? Did you generate this with pip-compile?

Reply:
Did `pip-compile requirements/linting.in -o requirements/linting.txt` exactly

Reply:
> Are you sure `tomli` should be removed here? Did you generate this with pip-compile?

Apologies, you were right indeed. I ran `pip-compile` on `3.11` initially. 
Now ran the above command (`pip-compile requirements/linting.in -o requirements/linting.txt`) in `3.10` and `tomli` is indeed kept as is (which makes sense considering it was added from `black`/`mypy`). Fixed in latest commit, sorry about that

Reply:
No problem, easy mistake to make. Would be wonderful if pip-compile could know what versions are required.